File: 2015/9/15Day9.py
import os
import numpy as np

D9_file = 'Day9_routes.txt'
D9_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), D9_file)

with open(D9_file_path) as file:
    routes_list = file.read()

routes_list = routes_list.splitlines()

print(routes_list)

File: 2015/9/15Day9.R
# Load necessary library
library(stringr)

# Define the file name and path
D9_file <- 'Day9_routes.txt'
D9_file_path <- file.path(dirname(rstudioapi::getActiveDocumentContext()$path), D9_file)

# Read the file
routes_list <- readLines(D9_file_path)

# Print the routes list
print(routes_list[2])
